src/libasr/codegen/asr_to_fortran.cpp

```diff
@@ -400,7 +400,9 @@ class ASRToFortranVisitor : public ASR::BaseVisitor<ASRToFortranVisitor>
 
     // void visit_Associate(const ASR::Associate_t &x) {}
 
-    // void visit_Cycle(const ASR::Cycle_t &x) {}
+    void visit_Cycle(const ASR::Cycle_t &/*x*/) {
+        s = indent + "continue\n";
```

Comment:
I think this should be 
```suggestion
        s = indent + "cycle\n";
```